pyuvdata/uvdata/uvdata.py

```diff
@@ -5904,8 +5904,8 @@ def read_mwa_corr_fits(
         end_flag=2.0,
         flag_dc_offset=True,
         phase_to_pointing_center=False,
-        phase_data=None,
-        phase_center=None,
+        data_array_dtype=np.complex64,
```

Comment:
Why is this defaulted differently than the `read` method or the `read_mwa_corr_fits` method on the MWACorrFITS object?

Reply:
Good question. I think because I wasn't paying attention :)

I think given the raw data is single precision, I'd advocate for the default here and in the `MWACorrFITS.read_mwa_corr_fits` method to match that (`np.complex64`), but keep the default on the `UVData.read()` double-precision to avoid changing the current default on users. What are your thoughts?

Reply:
That sounds reasonable to me.